examples/src/main/java/org/apache/spark/examples/ml/JavaWord2VecExample.java

```diff
@@ -55,10 +56,14 @@ public static void main(String[] args) {
       .setOutputCol("result")
       .setVectorSize(3)
       .setMinCount(0);
+
     Word2VecModel model = word2Vec.fit(documentDF);
     Dataset<Row> result = model.transform(documentDF);
-    for (Row r : result.select("result").takeAsList(3)) {
-      System.out.println(r);
+
+    for (Row row : result.collectAsList()) {
+      java.util.List text = row.getList(0);
```

Comment:
Yeah, just saying it's also fully qualified here. It could have a generic bound too.